apps/src/applab/applab.js

```diff
@@ -772,9 +770,14 @@ Applab.init = function(config) {
           );
         }
       }
-
+    })
+    .then(() => {
       if (getStore().getState().pageConstants.widgetMode) {
-        Applab.runButtonClick();
+        dataLoadedPromise
```

Comment:
can you set `dataLoadedPromise = Promise.resolve()` if there are no tables to load? That way you don't need this ternary operator here, you can just do `dataLoadedPromise.then(...)`

Reply:
Good point. Thanks!